src/Card/Card.story.tsx

```diff
@@ -0,0 +1,98 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import { storiesOf } from "@storybook/react";
+import { Card, CardSection, CardSeperator } from "./Card";
+import { Button } from "../Button";
+import * as colors from "../colors";
+
+storiesOf("Card", module)
+  .add("default", () => (
```

Comment:
Create a story with no description and no title

Think of stories as having two purposes:

1. To develop (as you know)
2. To validate for visual regressions in chromatic (which we haven't added yet, but we will!)